src/graphcast_agent/waku_handling.rs

```diff
@@ -48,36 +44,31 @@ pub fn build_content_topics(
 /// Makes a filter subscription from content topics and optionally pubsub topic
 /// Strictly use the first of pubsub topics as we assume radios only listen to one network (pubsub topic) at a time
 pub fn content_filter_subscription(
-    pubsub_topic: &WakuPubSubTopic,
+    pubsub_topic: String,
```

Comment:
`WakuPubSubTopic` seems like it was updated to be an alias for String, would you be able to keep the type as `WakuPubSubTopic` for clarity?